src/openfermion/utils/_lcu_util.py

```diff
@@ -17,6 +17,24 @@
 import math
 
 
+def lambda_norm(diagonal_operator):
+    """Computes the lambda norm relevant to LCU algorithms.
+
+    Args:
+        diagonal_operator: instance of DiagonalCoulombHamiltonian.
+
+    Returns:
+        lambda_norm: A float giving the lambda norm.
+    """
+    lambda_norm = 0.
+    n_qubits = diagonal_operator.one_body.shape[0]
+    for p in range(n_qubits):
+        for q in range(n_qubits):
+            lambda_norm += 0.75 * abs(diagonal_operator.two_body[p, q])
```

Comment:
Explain the 0.75 in a comment! If it's from what I think it is you can do the same improvement in certain cases of the next line too :)

Reply:
(This may be the reason for your test failures! But it's a little tricky to tell with a random test.)